pkg/pkgmgr/dpkg.go

```diff
@@ -459,115 +477,96 @@ func (dm *dpkgManager) unpackAndMergeUpdates(ctx context.Context, updates unvers
 			})).Root()
 	}
 
+	updated = updated.Run(buildkit.Sh("mkdir -p /tmp/debian-rootfs/var/lib/dpkg")).Root()
```

Comment:
```suggestion
	updated = updated.File(llb.Mkdir("/tmp/debian-rootfs/var/lib/dpkg", llb.WIthParents(true))
```

Wrote this manually here so there may be a mistake but should do this with LLB instead of spinning up a shell.